shared/src/persistence/dynamo/users/createPetitionerUserRecords.js

```diff
@@ -0,0 +1,36 @@
+const client = require('../../dynamodbClientService');
+
+const createPetitionerUserRecords = async ({
+  applicationContext,
+  user,
+  userId,
+}) => {
+  delete user.password;
+
+  await client.put({
+    Item: {
+      pk: `user|${userId}`,
+      sk: `user|${userId}`,
+      ...user,
+      userId,
+    },
+    applicationContext,
+  });
+
+  const formattedEmail = user.email.toLowerCase().trim();
```

Comment:
Have to think on this one. Our Cognito configuration currently allows for mixed case emails. If another email address with another case comes in it would destroy this. 🤔

Reply:
Oof, that's prickly! I think the spec generally allows for local boxes to be treated as case sensitive (hence the option in Cognito) - where convention often tells us emails are case-insensitive. I wonder if we could determine if any currently exist that "match" without case sensitivity, and then you could just decide what kind of world you want to run. 😛 

That's assuming you can even update that Cognito setting at this point. Should we look into just removing this `toLowerCase` method for now to keep this simple?

Reply:
Looks like we also fetch them as lower case here: 
https://github.com/ustaxcourt/ef-cms/blob/staging/shared/src/persistence/dynamo/users/getUserByEmail.js#L11

But I don't see where we're using that method (just noting it in case we want to change this behavior).

Reply:
I'm fairly certain we cannot change the Cognito setting at this point. 

I'm going to add some tickets to address this by letting the application take over the registration of users (ensuring lowercase) away from Cognito UI. 

It looks like this function is only called by a function to set up seed data. I think we are good here (for now).